spec/root-apps/systemjs.spec.js

```diff
@@ -17,10 +17,24 @@ import multipleLifecycleFunctions from 'spec/child-apps/multiple-lifecycle-funct
 import unmountRejects from 'spec/child-apps/unmount-rejects/unmount-rejects.spec.js';
 import unmountTimesOut from 'spec/child-apps/unmount-times-out/unmount-times-out.spec.js';
 import unmountTimesOutDies from 'spec/child-apps/unmount-times-out-dies/unmount-times-out-dies.spec.js';
+import usesLoader from 'spec/child-apps/uses-loader/uses-loader.spec.js';
 import navigateToUrlTests from 'spec/apis/navigate-to-url.spec.js';
 
 describe("SystemJS loader :", () => {
 	beforeAll(done => {
+		const ogSystemNormalize = System.normalize;
+
+		System.normalize = function(name, ...rest) {
```

Comment:
Haha a couple of hours before I got both webpack and systemjs using the same test files and still working....

All in the name of testing!